packages/fxa-payments-server/server/lib/amplitude.test.js

```diff
@@ -0,0 +1,26 @@
+const amplitude = require('./amplitude.js');
+const assert = require('chai').assert;
+
+const amplitudeConfig = {
+  disabled: false,
+};
+
+const testEvents = ['amplitude.sub.relyingPartyView'];
+// TODO: mock out logger to examine the fired log things.
+
+// TODO: test the snakeCase conversions: amplitude.sub.relyingPartyView -> amplitude.fxa_subscribe.relying_party_view
+
+describe('amplitude', () => {
+  describe('transforms and logs valid data for all planned ping formats', () => {
+    // TODO enumerate all planned pings, ensure they validate.
+    it('needs a placeholder or will fail', () => {
+      assert(true);
```

Comment:
This test isn't actually doing anything.

Reply:
Yeah, mocha barfs if there's an empty test, that was a placeholder to use while getting the build step green.